extensions/amp-list/0.1/amp-list.js

```diff
@@ -399,6 +389,48 @@ export class AmpList extends AMP.BaseElement {
     });
   }
 
+  /**
+   * @param {!Array} items
```

Comment:
Should the non-parameterized `{!Array}` types be parameterized to `!Array<?JsonObject>`?

Reply:
Hmm actually I think Array is itself a `JsonObject` so it might work to only pass around `JsonObject`

Reply:
I changed this to `!Array<?JsonObject>` because we want to do array operations on it.